client/src/app/pages/[lang]/articles/(articles-list).page.ts

```diff
@@ -0,0 +1,37 @@
+import { injectContentFiles } from '@analogjs/content';
+import { AsyncPipe, NgFor } from '@angular/common';
+import { Component, ChangeDetectionStrategy, inject } from '@angular/core';
+import { RouterLink } from '@angular/router';
+
+import { ArticleAttributes } from '@benjilegnard/resum/shared/model';
+import { TranslocoDirective, TranslocoService } from '@ngneat/transloco';
+
+@Component({
+  selector: 'bl-articles',
+  template: `
+    <ng-container *transloco="let t; read: 'articles'">
+      <h2>{{ t('title') }}</h2>
+      <ul>
+        <li *ngFor="let article of articles">
+          <a [routerLink]="article.slug">{{ article.attributes.title }}</a>
+        </li>
+      </ul>
+    </ng-container>
+  `,
+  styles: [``],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [RouterLink, NgFor, AsyncPipe, TranslocoDirective],
```

Comment:
__issue__: AsyncPipe pas utilisée
```suggestion
  imports: [RouterLink, NgFor, TranslocoDirective],
```